test_runner/src/main/kotlin/ftl/run/platform/RunAndroidTests.kt

```diff
@@ -78,6 +78,10 @@ internal suspend fun runAndroidTests(args: AndroidArgs): TestResult = coroutineS
     )
 }
 
+private fun String.createGcsPath(contextIndex: Int, runIndex: Int) =
+    if (runIndex == 0) "$this/matrix_$contextIndex/"
+    else "$this/matrix_${contextIndex}_$runIndex/"
```

Comment:
```suggestion
    else "$this/matrix_$contextIndex_$runIndex/"
```

Reply:
Without put ```contextIndex``` in ```{ }``` IntelliJ thinks ```_``` is a part of value name, so build is failing. 😄